### PR TITLE
feat: add aliases to all available commands

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
enhancement

**Did you add tests for your changes?**
Nope

**If relevant, did you update the documentation?**
Nope

**Summary**
- Added aliases for all available commands.
- Updated the help information to show up command aliases.
Screenshot 2020-06-23 at 4 39 27 PM

**Does this PR introduce a breaking change?**
Nope

**Other information**
N/A

### CODE REVIEW

**Comment by evilebottnawi** [comment]
Let's do for each command small alias (first letter of work):
- init | i
- migrate | m
- loader | l
- etc

File: packages/webpack-cli/lib/utils/cli-flags.js [hunk 1]
```diff
@@ -63,6 +65,7 @@ module.exports = {
         },
         {
             name: 'serve',
+            alias: 's',
             scope: 'external',
```
Comment (snitin315) [inline packages/webpack-cli/lib/utils/cli-flags.js:69]: Alias `s` is already occupied by `sourcemap`, should be removed from there. 
https://github.com/webpack/webpack-cli/blob/ac2e52c2ef8aa74aa795476be94ce7c968b8d1c5/packages/webpack-cli/lib/utils/cli-flags.js#L168-L175

Comment (jamesgeorge007) [reply packages/webpack-cli/lib/utils/cli-flags.js:69]: `sourcemap` is a `cli-flag` as oppoesd to `serve` which is a command.
`webpack s` vs `webpack -s <value>`

Comment (snitin315) [reply packages/webpack-cli/lib/utils/cli-flags.js:69]: oh yes, my bad :-)

**Pull request merged**